scripts/prebuild.js

```diff
@@ -50,10 +50,20 @@ async function checkPrebuild() {
       const key = e[0];
       const value = e[1];
 
+      let env_url = process.env.VERCEL_BRANCH_URL;
+      if (env_url && !env_url.startsWith('http')) {
```

Comment:
Technically possible the domain starts with `http`, it should instead be a regex or (!env_url.startsWith('http://') || !env_url.startsWith('https://'))